libqtile/backend/wayland/core.py

```diff
@@ -122,11 +122,11 @@ def log_cb(importance: int, formatted_str: ffi.CData) -> None:
     """Callback that logs the string at the given level"""
     log_str = ffi.string(formatted_str).decode()
     if importance == lib.WLR_ERROR:
-        qw_logger.error(log_str)
+        logger.error(log_str)
```

Comment:
This probably shouldn't be part of this PR but I don't understand why this isn't the default position.

Reply:
fine by me, let's tackle fixing qw_logger later

Reply:
I'll remove this from the PR for now.